src/dmd/target.d

```diff
@@ -278,6 +286,21 @@ extern (C++) struct Target
             assert(0, "unknown environment");
     }
 
+    ObjectFormat objectFormat() const pure
```

Comment:
Isn't this just duplicating what `__traits(getTargetInfo, "objectFormat")` does?